core/src/main/java/org/eclipse/hono/util/CommandConstants.java

```diff
@@ -0,0 +1,23 @@
+package org.eclipse.hono.util;
+
+/**
+ * Commands utility methods used throughout the Command and Control API.
+ */
+public class CommandConstants {
+
+    /**
+     * Empty default constructor.
+     */
+    protected CommandConstants () {
+    }
+
+    /**
+     * The name of the Command and Control API. endpoint.
+     */
+    public static final String COMMAND_ENDPOINT = "control";
```

Comment:
it feels strange that the *command endpoint* name is `control`

Reply:
I took this from the old API definition. First I also thought it is strange but to think that the commands  are send to a control endpoint sounds not so bad.